Packages/ReachySimulator/Scripts/ReachyController.cs

```diff
@@ -269,7 +269,7 @@ public void HandleCommand(Dictionary<JointId, float> commands)
                 if (motorName == "neck_yaw")
                 {
                     containNeckCommand = true;
-                    headOrientation[2] = kvp.Value;
+                    headOrientation[2] = -kvp.Value;
```

Comment:
There is a minus for neck_yaw in jointserviceserver and also here. We should remove both?

Reply:
No, we want to keep both:
- the first one is the value returned for the joint when using GetJointState, which needs a minus to be consistent with the value returned by a real robot
- the second one is the value used to update the head rotation in the scene, which needs to be opposite to the previous one so that the head rotation in the simu is consistent with the operator head movement